docs/tutorials/getting-started.ipynb

```diff
@@ -0,0 +1,469 @@
+    "Dimension labels are also used by other operations, clearly expressing the intent:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3129988f-e5ea-44b0-9227-894b20b89ac5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var.mean('location')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e22a643-6a13-4454-8e0c-fb49c290cc9d",
+   "metadata": {},
+   "source": [
+    "## Arrays with coordinates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09a4ce84-7988-48b2-8216-916155d25938",
+   "metadata": {},
+   "source": [
+    "Variables can be enhanced by **coordinates**.\n",
+    "Each coordinate is also a variable.\n",
+    "A variable with associated coordinates is called **data array**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42dc50b7-fc05-43e6-a816-621c12019a2a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "location = sc.linspace('location', 1.2, 1.3, num=4, unit='mm')\n",
+    "time = sc.linspace('time', 1.2, 1.3, num=3, unit='s')\n",
+    "data = sc.array(dims=['location', 'time'], unit='K', values=np.arange(12).reshape(4, 3))\n",
+    "da = sc.DataArray(data, coords={'location': location, 'time': time})\n",
+    "da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a3c32e4-0351-4a72-967d-188ba4b4c9fe",
+   "metadata": {},
+   "source": [
+    "Data array coordinates protect against operations between incompatible data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74945850-8ae2-40a5-93ef-fed0fb128847",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "other = da.copy()  # Copy and modify, for illustration purposes\n",
+    "other.coords['time'][-1] += 0.1 * sc.Unit('s')\n",
+    "da + other  # will raise an exception"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "626c40fa-5af3-457a-a529-dfe52d96054e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da['location', 0:2] - da.mean('location')  # ok, mean over location drops location coord"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2d0969e-c32e-473e-9953-f45b87236040",
+   "metadata": {},
+   "source": [
+    "Another application of coordinates is [label-based indexing](https://scipp.github.io/user-guide/slicing.html#Label-based-indexing):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f32451e-20e5-4e27-b823-a9c7b2d7097b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = 1.22 * sc.Unit('s')\n",
+    "stop = 1.4 * sc.Unit('s')\n",
+    "da['time', start:stop]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2802cfc9-4599-4602-8c9b-20018b0ed807",
+   "metadata": {},
+   "source": [
+    "In addition to coordinates, data arrays also provide a dictionary of attributes and a dictionary of masks."
```

Comment:
I think we should either remove this sentence, or add explicit examples for illustration.